mkt/api/tests/__init__.py

```diff
@@ -19,7 +21,8 @@ def _allowed_verbs(self, url, allowed):
             if verb in allowed:
                 continue
             try:
-                res = getattr(self.client, verb)(url)
+                with transaction.atomic():
```

Comment:
that's too weird to silence, IMHO. we should never need to add `transaction.atomic()` calls in tests.